verify.md

```diff
@@ -1,47 +1,47 @@
-## How to check the files for safety/integrity and authenticity:
+## How to check ours (but also others from different projects) files for safety/integrity and authenticity:
 
 The PDF and ODT files in this guide are cryptographically signed using GPG and [Minisign](https://jedisct1.github.io/minisign). Their integrity can be verified with the published SHA256 Checksum hashes on this website. SHA256 checksums of all the PDF and ODT files are available here in the [sha256sum.txt](sha256sum.txt) file. SHA256 Checksums, signatures, and VirusTotal ("VT") checks of the releases files (containing the whole repository) are available within the latest release information at <https://github.com/Anon-Planet/thgtoa/releases/latest> which will be available as soon as we have a stable release.
 
 The GPG signatures for each PDF and ODT files are available here:
-- PDF (Light Theme) Main and Mirrors: [guide.pdf.asc](guide.pdf.asc)
-- ODT Main and Mirrors: [guide.odt.asc](guide.odt.asc)
+- <del>PDF (Light Theme) Main and Mirrors: [guide.pdf.asc](guide.pdf.asc)</del> (Currently unavailable)
+- <del>ODT Main and Mirrors: [guide.odt.asc](guide.odt.asc)</del> (Currently unavailable)
 
 The Minisign signatures for each PDF and ODT files are available here:
-- PDF (Light Theme) Main and Mirrors: [guide.pdf.minisig](guide.pdf.minisig)
-- ODT Main and Mirrors: [guide.odt.minisig](guide.odt.minisig)
+- <del>PDF (Light Theme) Main and Mirrors: [guide.pdf.minisig](guide.pdf.minisig)</del> (Currently unavailable)
+- <del>ODT Main and Mirrors: [guide.odt.minisig](guide.odt.minisig)</del> (Currently unavailable)
 
 ### How to check the integrity of the files using the SHA256 Checksums:
 
-Please do the following:
+First get the hash of your local file by following these steps for your OS:
 
 Windows:
 - From a command prompt, run ```certutil -hashfile filename.txt sha256```
-- Compare the result with the hash in the online checksum files. They should match.
+- Compare the obtained hash result of your local file to the online files published hash. They should match.
 
 MacOS:
 - From a terminal, run ```shasum -a 256 /full/path/to/your/file```
-- Compare the result with the hash in the online checksum files. They should match.
+- Compare the obtained hash result of your local file to the online files published hash. They should match.
 
 Linux:
 - From a terminal, run ```sha256sum /full/path/to/your/file```
-- Compare the result with the hash in the online checksum files. They should match.
+- Compare the obtained hash result of your local file to the online files published hash. They should match.
 
-All commits and releases on this repository are cryptographically signed and verified using the same GPG key.
+All commits and releases on this repository are cryptographically signed, and verified using the same GPG key.
 
-**Check for the "Verified" tags on each commit or release.**
+**Do check for the "Verified" tags on each commit or release.**
```

Comment:
"Do check" implies you mean to add "Don't do this though" and makes it sound odd.